src/pages/myGeotab/apiReference/methodsCodeSamples.ts

```diff
@@ -0,0 +1,331 @@
+import { CodeSamples } from "../../../components/CodeSamplesContainer";
+
+export const methodsCodeSamples: Record<string, CodeSamples> = {
+  GetTimeZones: {
+    javascript: "",
+    csharp: `await api.CallAsync<List<Geotab.Checkmate.ObjectModel.TimeZoneInfo>>("GetTimeZones");`,
+    python: "api.call('GetTimeZones')",
+    java: "/* Work in progress - Coming soon */",
+  },
+  GetVersion: {
+    javascript: "",
+    csharp: `await api.CallAsync<string>("GetVersion");`,
+    python: "api.call('GetVersion')",
+    java: "/* Work in progress - Coming soon */",
+  },
+  GetVersionInformation: {
+    javascript: "",
+    csharp: `await api.CallAsync<object>("GetVersionInformation");`,
+    python: "api.call('GetVersionInformation')",
+    java: "/* Work in progress - Coming soon */",
+  },
+  OptimizeWaypoints: {
+    javascript: "",
+    csharp: `
+    var waypointsList = new List<Waypoint>
```

Comment:
Tabbing needs to be fixed, missing two waypoints, and missing descriptions for each waypoint.

```suggestion
    IList<Waypoint> waypointsList = new List<Waypoint>
```